package/MDAnalysis/coordinates/PDB.py

```diff
@@ -835,7 +836,9 @@ def _write_pdb_header(self):
 
         u = self.obj.universe
         self.HEADER(u.trajectory)
```

Comment:
#741 says that both the `TITLE` and `HEADER` record should only appear once in the PDB.  This includes all PDB records that should only be listed once. So this whole `_write_pdb_header` function should only be called once per written pdb.